profiles/insights/proxy/main.go

```diff
@@ -1,193 +1,195 @@
 package main
 
 import (
-	"encoding/base64"
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
-	"log"
-	"math/rand"
-	"net/http"
-	"net/url"
-	"os"
-	"regexp"
-	"strings"
-	"time"
+    "encoding/base64"
```

Comment:
have you used ` go fmt` or it is your text editor fixing indentation?

Reply:
AFAIK in Golang the standard is to use tabs instead of spaces, and that is what `go fmt` sets by default.